docs/index.rst

```diff
@@ -1,9 +1,16 @@
-Hypothesis
-==========
+Welcome to Hypothesis Web Service’s documentation!
```

Comment:
The capitals are because this one's a title though, but _the_ shouldn't be missing